controllers/amphoracontroller_controller.go

```diff
@@ -463,19 +442,24 @@ func (r *OctaviaAmphoraControllerReconciler) generateServiceConfigMaps(
 	if err != nil {
 		return err
 	}
+
+	parentOctaviaName := amphoracontrollers.GetOwningOctaviaControllerName(
+		instance)
+	serverCAPassSecretName := fmt.Sprintf("%s-ca-passphrase", parentOctaviaName)
 	caPassSecret, _, err := secret.GetSecret(
-		ctx, helper, instance.Spec.CAKeyPassphraseSecret, instance.Namespace)
+		ctx, helper, serverCAPassSecretName, instance.Namespace)
 	if err != nil {
 		return err
 	}
```

Comment:
It might be beneficial to set a condition to an error state here so that, if the secret is missing, the CLI shows it via `oc get amphoracontroller` and it's not just hidden in the operator logs. i.e. similar to the pattern here [1].  Also I wonder if we should check if `parentOctaviaName` is the empty string...but I guess that will get surfaced via the missing secret a few lines later.

[1] https://github.com/openstack-k8s-operators/octavia-operator/blob/e0743a7e88c841c3a615d36385487487bc1afc08/controllers/amphoracontroller_controller.go#L215-L232